Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,433 @@
+        m_tabWidget->setContentsMargins(0, 0, 0, 0);
+        
+        m_tabWidget->setObjectName("createGemTab");
+        m_tabWidget->addTab(CreateGemSetupScrollArea(), tr("1. Gem Setup"));
+        m_tabWidget->addTab(CreateGemDetailsScrollArea(), tr("2. Gem Details"));
+        m_tabWidget->addTab(CreateGemCreatorScrollArea(), tr("3. Creator Details"));
+        m_tabWidget->tabBar()->setEnabled(false);
+        vLayout->addWidget(m_tabWidget);
+
+        QFrame* footerFrame = new QFrame(this);
+        footerFrame->setObjectName("createAGemFooter");
+        m_backNextButtons = new QDialogButtonBox(this);
+        m_backNextButtons->setObjectName("footer");
+        QVBoxLayout* footerLayout = new QVBoxLayout();
+        footerLayout->setContentsMargins(0, 0, 0, 0);
+        footerFrame->setLayout(footerLayout);
+        footerLayout->addWidget(m_backNextButtons);
+        vLayout->addWidget(footerFrame);
+
+        m_backButton = m_backNextButtons->addButton(tr("Back"), QDialogButtonBox::RejectRole);
+        m_backButton->setProperty("secondary", true);
+        m_nextButton = m_backNextButtons->addButton(tr("Next"), QDialogButtonBox::ApplyRole);
+
+        connect(m_tabWidget, &TabWidget::currentChanged, this, &CreateGem::UpdateNextButtonToCreate);
+        connect(m_backButton, &QPushButton::clicked, this, &CreateGem::HandleBackButton);
+        connect(m_nextButton, &QPushButton::clicked, this, &CreateGem::HandleNextButton);
+
+        setObjectName("createAGemBody");
+        setLayout(vLayout);
+    }
+
+    void CreateGem::LoadButtonsFromGemTemplatePaths(QVBoxLayout* gemSetupLayout)
+    {
+        m_radioButtonGroup = new QButtonGroup();
+
+        auto templatesResult = PythonBindingsInterface::Get()->GetGemTemplates();
+        if (templatesResult.IsSuccess() && !templatesResult.GetValue().isEmpty())
+        {
+            m_gemTemplates = templatesResult.GetValue();
+            for (int index = 0; index < m_gemTemplates.size(); ++index)
+            {
+                ProjectTemplateInfo gemTemplate = m_gemTemplates.at(index);
+
+                QRadioButton* button = new QRadioButton(gemTemplate.m_displayName);
+                button->setObjectName("createAGem");
+                m_radioButtonGroup->addButton(button, index);
+
+                QLabel* buttonSubtext = new QLabel();
+                buttonSubtext->setObjectName("createAGemRadioButtonSubtext");
+                buttonSubtext->setText(gemTemplate.m_summary);
+                if (index == 0)
+                {
+                    button->click();
+                }
+                gemSetupLayout->addWidget(button);
+                gemSetupLayout->addWidget(buttonSubtext);
+            }
+        }
+    }
+
+    QScrollArea* CreateGem::CreateGemSetupScrollArea()
+    {
+        QScrollArea* gemSetupScrollArea = new QScrollArea();
+        gemSetupScrollArea->setWidgetResizable(true);
+        gemSetupScrollArea->setMinimumWidth(660);
```

Comment:
we could probably move this to qss by adding a `min-width: 660` entry to `#createAGemRightPane`.  I think there are multiple instances of this.